configure.ac

```diff
@@ -26,6 +26,14 @@ AC_ARG_WITH([deadline],
 
 AM_CONDITIONAL([SET_DLSCHED], [test x$with_deadline != xno && test !HAVE_SCHED_SETATTR])
 
+AC_ARG_WITH([uclamp],
+	[AS_HELP_STRING([--with-uclamp],
+		[Add support for SCHEDA_UCLAMP])],
+	[AC_DEFINE([UCLAMP], [1], [Define if you have SCHED_UCLAMP support])],
+	[with_uclamp=no])
+
+AM_CONDITIONAL([SET_UCLAMP], [test x$with_uclamp != xno && test !HAVE_SCHED_SETATTR])
```

Comment:
I think this (and the with_deadline) options are not necessary. If the kernel doesn't have the support you should get an error and bailout. Why complicate the building process if we can dynamically manage this?

Reply:
Done